.SysId/sysid.json

```diff
@@ -0,0 +1,11 @@
+{
+  "SysId": {
+    "Analysis Type": "General Mechanism",
+    "AnalyzerJSONLocation": "C:\\Users\\team467\\Documents\\GitHub\\Robot-Code\\sysid\\drivetrain\\sysid_data20220121-184451.json",
```

Comment:
Does this need to vary by robot?

Reply:
this is just the last open file, and the characterizations are moved to folders for each robot